realm/build.gradle

```diff
@@ -138,3 +138,84 @@ tasks.preBuild {
     dependsOn compileAnnotations
     dependsOn buildApt
 }
+
+def examples = [
```

Comment:
Do we really want to enumerate all the projects we want to include. Wouldn't it be better to name the ones we *don't* want. I would assume that would be easier to maintain as the most common use case would be to include them?